mimic/rest/auth_api.py

```diff
@@ -124,6 +124,17 @@ def get_username(self, request, tenant_id):
         session = self.core.sessions.session_for_tenant_id(tenant_id)
         return json.dumps(dict(user=dict(id=session.username)))
 
+    @app.route('/v2.0/users/<string:userid>/OS-KSADM/credentials/RAX-KSKEY:apiKeyCredentials',
+               methods=['GET'])
+    def rax_kskey_apikeycredentials(self, request, userid):
+        """
+        Support, such as it is, for the apiKeysCredentials call.
+        """
+        apikey = '1337133713371337133713371337'
+        username = 'mimicuser'
```

Comment:
This state should really be stored on the `Session`, since that is already keyed by username, using the same idiom of "create it if there's nothing that matches already".  Hopefully it won't be much harder to do this?